docs/UserGuide.md

```diff
@@ -134,18 +135,33 @@ Curious about the various commands? Refer to the [Features](#features) below to
 
 ## Before Using PennyWise
 
+### What is an Entry
+
+Think of PennyWise as a budgeting diary that has infinite number of pages. You can write as much data as you want 
```

Comment:
Hmm perhaps we can just say personal budgeting assistant as Emily suggested? Infinite pages is not completely true since it is still limited by the amount of storage the user has. Also, our application seems to have already exceeded the capabilities of a budgeting diary, with the graphical display and all.

"
Think of PennyWise as a helpful personal budgeting assistant. Whenever you spend on something or make some money, all you need to do is tell PennyWise and PennyWise will help you log and save it! Not only that, PennyWise will help you analyse your data so you can easily get an overview of your overall expenditure or income.

However, in order to do this, PennyWise needs some data from you! Whenever you log an entry, whether expenditure or income, you need to tell PennyWise the following:
...what data is required...
"
Tried to make the tone a little more friendly too but edit as you see fit!

Reply:
Have altered it please take a look again!